activities/member-relations/collect-information.md

```diff
@@ -0,0 +1,35 @@
+
+---
+type: Guide
```

Comment:
You can add an `explains: ` here if you want to and if this is a guide

Reply:
made into resource